habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/util/DockerfileUtil.java

```diff
@@ -0,0 +1,104 @@
+package org.technologybrewery.habushu.util;
+
+import org.technologybrewery.habushu.HabushuException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+public class DockerfileUtil {
+    public static final String HABUSHU_FINAL_STAGE = "#HABUSHU_FINAL_STAGE";
+    public static final String HABUSHU_BUILDER_STAGE = "#HABUSHU_BUILDER_STAGE";
+    public static final String HABUSHU_COMMENT_START = " - HABUSHU GENERATED CODE (DO NOT MODIFY)";
+    public static final String HABUSHU_COMMENT_END = " - HABUSHU GENERATED CODE (END)";
+    public static final String REPLACE_WITH_SINGLE_REPO_PROJECT_DIR = "REPLACE_WITH_SINGLE_REPO_PROJECT_DIR";
+    public static final String ANCHOR_DIRECTORY = "ANCHOR_DIRECTORY";
+
+    /**
+     * Create the container stage content with given template, anchor directory and the single module's base directory
+     * @param template container stage template name
+     * @param anchorDirectory the anchor directory
+     * @param moduleBaseDir the module base directory
+     * @return
+     */
+    public static String getContainerStageContentFrom(String template, String anchorDirectory, String moduleBaseDir) {
```

Comment:
Would make more sense to call this method `setContainerStageContentFrom` or `createContainerStageContentFrom`.